include/dbus_monitor.hpp

```diff
@@ -105,8 +105,8 @@ inline int onPropertyUpdate(sd_bus_message* m, void* userdata,
 inline void requestRoutes(App& app)
 {
     BMCWEB_ROUTE(app, "/subscribe")
-        .privileges({{"Login"}})
         .websocket()
+        .privileges({{"Login"}})
```

Comment:
Why change this?

Reply:
if I don't change then WebSocket authentication will not happen for "/subscribe". as you know previously there was no check for the socket.
so this has to change for authentication